src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -403,10 +406,73 @@ private static bool ShouldSkip(Symbol symbol)
         {
             return symbol.IsImplicitlyDeclared ||
                 symbol.IsAccessor() ||
-                symbol is SynthesizedSimpleProgramEntryPointSymbol ||
-                symbol is SynthesizedRecordPropertySymbol;
+                symbol is SynthesizedSimpleProgramEntryPointSymbol;
+        }
+
+        private bool TryProcessRecordPropertyDocumentation(
+            SynthesizedRecordPropertySymbol recordPropertySymbol,
+            ImmutableArray<DocumentationCommentTriviaSyntax> docCommentNodes,
+            [NotNullWhen(true)] out string? withUnprocessedIncludes,
+            out ImmutableArray<CSharpSyntaxNode> includeElementNodes)
+        {
+            _cancellationToken.ThrowIfCancellationRequested();
+
+            if (getMatchingParamTags() is not { } paramTags)
```

Comment:
>getMatchingParamTags() [](http://example.com/codeflow?start=16&length=22)

nit: consider making `getMatchingParamTags` static and passing the relevant state explicitly (`docCommentNodes` and the property's name?)